policyengine_us/parameters/gov/ed/pell_grant/efc/dependent/asset_modification.yaml

```
@@ -0,0 +1,9 @@
+description: The Education Department defines the percent of a dependent student's assets that is counted towards the expected family contribution.
```

Word other descriptions similarly, usually with `this` somewhere.
```suggestion
description: The Education Department counts this share of a dependent student's assets towards the expected family contribution.
```